src/components/TableStateContainer.tsx

```diff
@@ -23,9 +23,10 @@ interface Props {
   textSearchBox?: boolean; // default true
   newItemOnClick: () => void | null;
   queryCheckBox: {text: string, adaptUrlFunction: (url:string)=>string} | null;
+  scenarioIsNull?: boolean;
```

Comment:
@hoanphungt 
Hi Hoan,
I am not sure about this change.
Although this works,  it does not scale up well.
What if in another table we need another default url parameter? We need to keep adding props.
I think url parameters that are standard for a certain table should either be appended as string to "baseUrl" or be passed as a newly to be made props called something like "defaultUrlParameters". 
Please let me know what you think
kr Tom

Reply:
Ah you are right. I will improve it this way.